.packit.yaml

```diff
@@ -28,13 +28,17 @@ jobs:
   - job: copr_build
     trigger: pull_request
     targets:
-      centos-stream-8:
-        additional_modules: "foreman:el8,nodejs:14"
+      rhel-8:
+        additional_modules: "foreman-devel:el8"
         additional_repos:
-          - http://koji.katello.org/releases/yum/foreman-nightly/el8/x86_64/
+          - http://yum.theforeman.org/releases/nightly/el8/x86_64/
           - http://yum.theforeman.org/plugins/nightly/el8/x86_64/
+      rhel-9:
+        additional_modules: "foreman-devel:el9"
```

Comment:
Same as with the other plugin: https://docs.theforeman.org/nightly/Installing_Server/index-foreman-el.html#repositories-el-9

Reply:
I can try and see what happens when I remove it, but this is exactly how other plugins are doing it and the packit build was also successful.

Reply:
Well, it still builds. So I guess, we can just drop the line :)